cmd/clusterctl/client/config/providers_client.go

```diff
@@ -44,11 +44,12 @@ const (
 	// Bootstrap providers
 	KubeadmBootstrapProviderName = "kubeadm"
 	TalosBootstrapProviderName   = "talos"
-	EKSBootstrapProviderName     = "eks"
+	AWSEKSBootstrapProviderName  = "aws-eks"
```

Comment:
when changing/adding new providers some CLI test should be fixed as well, see eg. https://github.com/kubernetes-sigs/cluster-api/pull/3621/files
feel free to ping me if you need some help in getting them passing

Reply:
These have been changed